### PR TITLE
Add gitter to readme

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Adds reference to new Gitter space and standardizes some formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ View the [latest release documentation](http://data-tools-docs.ibi-transit.com/e
 
 Note: `dev` branch docs (which refer to the default `branch` and are more up-to-date and accurate for most users) can be found [here](http://data-tools-docs.ibi-transit.com/en/dev/).
 
+## Getting in touch
+
+We have a Gitter chat room for the full TRANSIT-Data-Tools project where you can post questions and comments: https://matrix.to/#/#transit-data-tools:gitter.im.
```
Comment (miles-grant-ibigroup) [inline README.md]: Can you please add the badge instead

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Can you please add the badge instead? Like https://github.com/opentripplanner/otp-react-redux/pull/898/files

File: README.md (hunk 1 above)
Comment (philip-cline) [reply README.md]: How about badge and link? https://github.com/ibi-group/datatools-ui/pull/953/commits/a6442d802e35be8c4c642115cfa9da460fbf75d5

File: README.md [hunk 2]
```diff
@@ -12,6 +14,10 @@ View the [latest release documentation](http://data-tools-docs.ibi-transit.com/e
 
 Note: `dev` branch docs (which refer to the default `branch` and are more up-to-date and accurate for most users) can be found [here](http://data-tools-docs.ibi-transit.com/en/dev/).
 
+## Getting in touch
+
+We have a Gitter [space](https://matrix.to/#/#transit-data-tools:gitter.im) for the full TRANSIT-Data-Tools project where you can post questions and comments.
```
Comment (miles-grant-ibigroup) [inline README.md:19]: Can we replace the link with the `gitter.im` one?

**Review by miles-grant-ibigroup — APPROVED** [review]
Copy reads great pending link change

File: README.md (hunk 2 above)
Comment (daniel-heppner-ibigroup) [reply README.md:19]: plz make this the same link

**Review by daniel-heppner-ibigroup — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (philip-cline) [reply README.md:19]: Same link but matrix. done.

**Pull request merged**